android/src/main/kotlin/com/google/maps/flutter/navigation/GoogleMapsAutoMapView.kt

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.maps.flutter.navigation
+
+import android.view.View
+import com.google.android.gms.maps.GoogleMap
+import com.google.android.gms.maps.GoogleMapOptions
+import com.google.android.libraries.navigation.NavigationViewForAuto
+import io.flutter.plugin.platform.PlatformView
+
+class GoogleMapsAutoMapView
+internal constructor(
+  mapOptions: GoogleMapOptions,
+  private val viewRegistry: GoogleMapsViewRegistry,
+  imageRegistry: ImageRegistry,
+  private val mapView: NavigationViewForAuto,
+  private val map: GoogleMap,
+) : PlatformView, GoogleMapsBaseMapView(null, mapOptions, null, imageRegistry) {
```

Comment:
Why this extends `PlatformView` as this is not created by `Flutter`.
Also remove unnecessary overrides

Reply:
Leftovers, fixed.